README.rst

```diff
@@ -46,17 +50,17 @@ choose appropriate downloads for the specified target platforms,
 and generate a flatpak-builder build module.
 The module, if included in a flatpak-builder build manifest,
 will install the required packages using pip.
-
 The commandline option to define target platforms uses the format ``<pythonversion>-<architecture>``.
 To learn more about available commandline options,
 run ``req2flatpak.py --help``.
 
-Programmatic usage is also possible.
-This means you can invoke functionality from req2flatpak in your own python script,
-allowing you to tweak the desired behavior in many ways.
-The `documentation <https://johannesjh.github.io/req2flatpak/>`__
-describes req2flatpak's python api and includes code examples
-to help you get started quickly.
+Note that programmatic usage of req2flatpakis also possible.
+This means you can write a python script to invoke req2flatpak,
+which allows to tweak the desired behavior in many ways.
+
+Go read the `documentation <https://johannesjh.github.io/req2flatpak/>`__
+to learn more about req2flatpak's commandline and python APIs.
```

Comment:
"and Python APIs"  (with a uppercase P).  Here and throughout, unless its referring to a command.  (Seems very minor, feel free to ignore!)